src/backend.h

```diff
@@ -107,6 +108,11 @@ namespace librealsense
             {
                 return std::chrono::duration<double, std::milli>(std::chrono::system_clock::now().time_since_epoch()).count();
             }
+
+            rs2_time_t get_time_ns() const override
+            {
+                return std::chrono::duration<double, std::nano>(std::chrono::system_clock::now().time_since_epoch()).count();
+            }
```

Comment:
Can we drop `get_time_ns` now that we are back to `ms`?

Reply:
Updated